README.md

```diff
@@ -89,11 +89,10 @@ __NOTE:__ The bug tracker is _not_ for discussions. Please use our [official cha
 
 ## Contributing  [![prwelcome.badge]](https://reactos.org/wiki/Commiting_Changes)
 
-We are always looking for developers! Check [how to contribute](CONTRIBUTING.md) if you are willing to participate.
+We are always looking for developers! Check [how to contribute](CONTRIBUTING.md) if you are willing to participate.</br></br>
+__Legal notice__: If you have seen proprietary Microsoft Windows source code (including but not limited to the leaked Windows NT 3.5, NT 4, 2000 source code and the Windows Research Kernel), your contribution won't be accepted because of potential copyright violation.
```

Comment:
Don't use `<br>` hacks, just two normal line breaks do the thing.
```suggestion
We are always looking for developers! Check [how to contribute](CONTRIBUTING.md) if you are willing to participate.

__Legal notice__: If you have seen proprietary Microsoft Windows source code (including but not limited to the leaked Windows NT 3.5, NT 4, 2000 source code and the Windows Research Kernel), your contribution won't be accepted because of potential copyright violation.
```